Nodejs/Product/Nodejs/Debugger/Commands/SetBreakpointCommand.cs

```diff
@@ -61,12 +61,9 @@ public SetBreakpointCommand(int id, NodeModule module, NodeBreakpoint breakpoint
             if (_module != null) {
                 _arguments["type"] = "scriptId";
                 _arguments["target"] = _module.Id;
-            } else if (remote) {
+            } else {
                 _arguments["type"] = "scriptRegExp";
                 _arguments["target"] = GetCaseInsensitiveRegex(_position.FileName);
```

Comment:
Now that I look more closely at `GetCaseInsensitiveRegex`, it only seems to use the filename in the regexp. That's odd,  since, in my understanding, with a project like:

```
a ->
    index.js
b ->
   index.js
```

You could not set a breakpoint in both of the `index.js` files.

Not sure this is proper fix given that, but perhaps a different regex can be used instead for local files.